scipy/interpolate/_interpolate.py

```diff
@@ -93,11 +93,42 @@ def lagrange(x, w):
 # !! found, get rid of it!
 
 
+dep_mesg = """\
+`interp2d` is deprecated in SciPy 1.10 and will be removed in SciPy 1.12.0. 
+
+For legacy code, nearly bug-for-bug compatible replacements are
+`RectBivariateSpline` on regular grids, and `bisplrep`/`bisplev` for
+scattered 2D data.
+
+In new code, for regular grids use `RegularGridInterpolator` instead.
+For scattered data, prefer `LinearNDInterpolator` or
+`CloughTocher2DInterpolator`.
+
+For more details see
+`https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff`
+"""
```

Comment:
```suggestion
dep_mesg = """\
'interp2d' is deprecated in SciPy 1.10 and will be removed in SciPy 1.12.0. 

For legacy code, nearly bug-for-bug compatible replacements are
'RectBivariateSpline' on regular grids, and 'bisplrep'/'bisplev' for
scattered 2D data.

In new code, for regular grids use 'RegularGridInterpolator' instead.
For scattered data, prefer 'LinearNDInterpolator' or
'CloughTocher2DInterpolator'.

For more details see
'https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff'
"""
```
Minor enough to ignore if you want but in other deprecation reviews there is has been a preference for single quotes over backticks as they are easier to see in the terminal

Reply:
Ugh. Neither passes the "speck on a monitor"  test for me, but if there's a preference, so be it